TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -372,6 +372,15 @@ public override bool Visit(DataDefinition dataDefinition)
             return true;
         }
 
+        public override bool Visit(SourceComputer sourceComputer)
+        {
+            if (sourceComputer.Lines.Any(l => (l as CodeElementsLine)?.ScanState.WithDebuggingMode == true))
+            {
+                DiagnosticUtils.AddError(sourceComputer, "Debugging mode is active", MessageCode.Warning);
```

Comment:
It'd be nice to position the diagnostic on the line that contains "DEBUGGING MODE" rather than  SOURCE-COMPUTER.
You can pass the token sourceComputer.DebuggingMode.Token to AddError method to do so.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/tree/94dd2d71d6dd982de8de0c257ff90aba9e55e698